### PR TITLE
Link to PDF version of course from first page

### PR DESCRIPTION
**Pull request by max-heller**
Re https://github.com/google/comprehensive-rust/pull/1805#pullrequestreview-1888804528

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ generics and error handling.
 > The latest version of the course can be found at
 > <https://google.github.io/comprehensive-rust/>. If you are reading somewhere
 > else, please check there for updates.
+>
+> The course is also available
+> [as a PDF](https://google.github.io/comprehensive-rust/comprehensive-rust.pdf).
```
Comment (mgeisler) [inline src/index.md:16]: I think a relative link will work better: it should be correct for all languages out of the box:

```suggestion
> [as a PDF](comprehensive-rust.pdf).
```

Comment (max-heller) [reply src/index.md:16]: I wasn't sure if that'd be an issue when the PDF isn't available (e.g., building the book locally with `mdbook build`). Is it fine if the link is broken in such cases?

Comment (mgeisler) [reply src/index.md:16]: Good question... I would say yes, building things locally is something people can (and should) do while developing the material, but the publishing workflow is the one that actually counts.

The `build.sh` script is a little step towards formalizing this. @sakex has been working on #1236, which will let `mdbook` do all the work for all the translations.

**Review by mgeisler — APPROVED** [review]
Thanks!

**Pull request merged**